src/contentsharecontroller/ContentShareMediaStreamBroker.ts

```diff
@@ -34,7 +34,7 @@ export default class ContentShareMediaStreamBroker implements MediaStreamBroker
   }
 
   releaseMediaStream(_mediaStreamToRelease: MediaStream): void {
-    this.logger.warn('release media stream called');
+    this.logger.info('release media stream called');
```

Comment:
Why do we need to this? Is this because `ReleaseMediaStreamTask` call `releaseMediaStream` for content share video?

Reply:
I think so. I was just adjusting log level because it didn't seem right.